backend/node_app/controllers/modularGameChangerController.js

```diff
@@ -177,10 +177,10 @@ class ModularGameChangerController {
 
 	async export(req, res) {
 		const userId = req.get('SSL_CLIENT_S_DN_CN');
-		const {cloneName, searchText, format, options} = req.body;
+		const {cloneName, searchText, classificationMarking, format, options} = req.body;
```

Comment:
Throw the classificationMarking into options. This way if there a clones that dont use it it wont break things.